selenium_tests/page.py

```diff
@@ -0,0 +1,32 @@
+""""""
+
+from selenium.webdriver.common.by import By
+
+
+class BasePage(object):
+    def __init__(self, browser):
+        self.browser = browser
+
+
+class LoginPage(BasePage):
```

Comment:
What's the intention of `BasePage` here, why not put everything in `LoginPage`? I'm not sure what functionality would go in `BasePage` in the future or what other instances of `BasePage` there would be